controllers/groupsController.js

```diff
@@ -0,0 +1,22 @@
+const groupModel = require("../models/groupModel");
+const { isValidString, isEmptyString } = require("../utils/validation");
+
+const list = async (req, res) => {
+  if (!isValidString(req.params.uid) || isEmptyString(req.params.uid)) {
+    return res.status(400).send({ message: "[InvalidUid] Error occured" });
+  }
+
+  try {
+    const { uid } = req.params;
+    const groupListResult = await groupModel
+      .find({ ownerUid: uid })
+      .populate("keywordIdList", "keyword");
+    res.status(200).json({ groupLength: groupListResult.length, groupListResult });
+  } catch (error) {
+    return res
+      .status(500)
+      .send({ message: "[ServerError] Error occured in 'groupsController.list'" });
+  }
```

Comment:
```suggestion
  } catch () {
    return res
      .status(500)
      .send({ message: "[ServerError] Error occured in 'groupsController.list'" });
  }
```

해당 코드 또한 사용하지 않는 `error` 는 삭제하면 될 것 같습니다.